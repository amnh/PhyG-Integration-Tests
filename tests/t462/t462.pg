--input t12.tre, swap(all), inputting prefasta file
set(seed:1634561640)
read(prefasta: "chel-prealigned.fas")
read(prefasta: "metazoa-aa-1.fas")
read(newick: "t12.tre")
swap(all)
report("t462.csv", data, overwrite)
report("t462.tre", newick, overwrite)