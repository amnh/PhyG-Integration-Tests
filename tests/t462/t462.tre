(Alentus:0.0,(Aportus:10.0,(Ilyanassa_obsoleta:0.0,(Scaphopods:9.0,((Littorina_littorea:1.0,Perotrochus_lucaya:0.0)HTU43:0.0,(Lottia_gigantea:5.0,((Vonones:10.0,(Amblypygid:12.0,(((Mastigoproctus:9.0,(Centruoides:7.0,(Hadrurus:11.0,(Lymnaea_stagnalis:0.0,Pauroctonus:0.0)HTU40:0.0)HTU35:10.0)HTU36:6.0)HTU37:4.0,(Hypochilus:1.0,((Crassostrea_virginica:16.0,(Aplysia_californica:2.0,(Gea:0.0,Siphonaria_pectinata:0.0)HTU33:1.0)HTU47:5.0)HTU46:13.0,(Helobdella_robusta:0.0,Thelichoris:0.0)HTU49:3.0)HTU45:1.0)HTU44:5.0)HTU34:2.0,(Chanbria:11.0,Limulus:7.0)HTU53:4.0)HTU42:3.0)HTU39:5.0)HTU38:8.0,((Mytilus_californianus:9.0,(((Americhernus:0.0,Venerupis_decussatus:0.0)HTU51:40.0,(Artemia:0.0,Venerupis_philippinarum:0.0)HTU54:15.0)HTU48:10.0,(Lingula_anatina:0.0,Rhiphicephalus:0.0)HTU62:8.0)HTU50:2.0)HTU41:21.0,(Thermobius:0.0,Yoldia_limatula:0.0)HTU59:10.0)HTU61:10.0)HTU52:1.0)HTU60:0.0)HTU58:1.0)HTU57:0.0)HTU56:10.0)HTU55:0.0)HTU32[573.0];
