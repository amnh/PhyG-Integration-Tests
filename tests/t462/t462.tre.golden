(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Centruoides:67.5,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:13.5)HTU37:10.0,(Scaphopods:82.0,((Lymnaea_stagnalis:76.0,Mastigoproctus:61.0)HTU42:9.0,(Siphonaria_pectinata:85.5,(Aplysia_californica:84.5,(Gea:82.5,(Hypochilus:57.5,Thelichoris:59.5)HTU62:4.5)HTU36:5.0)HTU55:3.5)HTU43:21.0)HTU41:11.0)HTU57:3.5)HTU40:7.0,((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:8.5)HTU38:14.0)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:8.5,((Scaphopods:78.5,(Centruoides:67.5,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:13.5)HTU37:6.0)HTU38:10.0,((Lymnaea_stagnalis:76.0,Mastigoproctus:61.0)HTU42:9.0,(Siphonaria_pectinata:85.5,(Aplysia_californica:84.5,(Gea:82.5,(Hypochilus:57.5,Thelichoris:59.5)HTU62:4.5)HTU36:5.0)HTU55:3.5)HTU43:21.0)HTU41:12.0)HTU57:7.0)HTU40:14.0)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:8.5,(Scaphopods:82.5,((Centruoides:67.5,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:13.5)HTU37:10.0,((Lymnaea_stagnalis:76.0,Mastigoproctus:61.0)HTU42:9.0,(Siphonaria_pectinata:85.5,(Aplysia_californica:84.5,(Gea:82.5,(Hypochilus:57.5,Thelichoris:59.5)HTU62:4.5)HTU36:5.0)HTU55:3.5)HTU43:21.0)HTU41:7.5)HTU38:9.5)HTU57:4.5)HTU40:14.0)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:9.0,(Scaphopods:82.5,((Mastigoproctus:67.0,(Lymnaea_stagnalis:76.0,(Centruoides:66.0,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:15.0)HTU37:5.5)HTU38:7.0)HTU42:5.0,(Siphonaria_pectinata:86.0,(Aplysia_californica:85.0,(Gea:82.5,(Hypochilus:57.5,Thelichoris:59.5)HTU62:4.5)HTU36:5.0)HTU55:3.5)HTU43:21.0)HTU41:12.0)HTU57:5.0)HTU40:13.5)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:9.0,(Scaphopods:82.5,(((Centruoides:66.0,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:15.0)HTU37:10.0,(Lymnaea_stagnalis:76.5,Mastigoproctus:61.0)HTU42:8.5)HTU38:5.0,(Siphonaria_pectinata:86.0,(Aplysia_californica:85.0,(Gea:82.5,(Hypochilus:57.5,Thelichoris:59.5)HTU62:4.5)HTU36:5.0)HTU55:3.5)HTU43:21.0)HTU41:12.0)HTU57:5.0)HTU40:13.5)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Centruoides:67.5,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:13.5)HTU37:10.0,(Scaphopods:82.0,((Lymnaea_stagnalis:76.0,Mastigoproctus:61.0)HTU42:9.0,(Siphonaria_pectinata:85.5,(Aplysia_californica:84.5,(Thelichoris:60.0,(Gea:83.0,Hypochilus:58.0)HTU36:3.5)HTU62:5.5)HTU55:3.5)HTU43:21.0)HTU41:11.0)HTU57:3.5)HTU40:7.0,((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:8.5)HTU38:14.0)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Centruoides:67.5,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:13.5)HTU37:10.0,(Scaphopods:82.0,((Lymnaea_stagnalis:76.0,Mastigoproctus:61.0)HTU42:9.0,(Siphonaria_pectinata:85.5,(Aplysia_californica:84.5,(Hypochilus:58.0,(Gea:83.0,Thelichoris:60.0)HTU62:3.5)HTU36:5.5)HTU55:3.5)HTU43:21.0)HTU41:11.0)HTU57:3.5)HTU40:7.0,((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:8.5)HTU38:14.0)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Centruoides:67.5,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:13.5)HTU37:10.0,(Scaphopods:82.0,((Lymnaea_stagnalis:76.0,Mastigoproctus:61.0)HTU42:9.0,(Siphonaria_pectinata:85.5,((Gea:83.0,Hypochilus:58.0)HTU36:3.5,(Aplysia_californica:85.0,Thelichoris:57.5)HTU62:5.0)HTU55:6.5)HTU43:21.0)HTU41:11.0)HTU57:3.5)HTU40:7.0,((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:8.5)HTU38:14.0)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Centruoides:67.5,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:13.5)HTU37:10.0,(Scaphopods:82.0,((Lymnaea_stagnalis:76.0,Mastigoproctus:61.0)HTU42:9.0,(Siphonaria_pectinata:85.5,(Thelichoris:60.0,(Aplysia_californica:85.0,(Gea:83.0,Hypochilus:58.0)HTU36:3.0)HTU55:3.0)HTU62:6.5)HTU43:21.0)HTU41:11.0)HTU57:3.5)HTU40:7.0,((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:8.5)HTU38:14.0)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
(Alentus:57.5,(Helobdella_robusta:82.0,(Aportus:68.0,((Thermobius:75.5,(Rhiphicephalus:74.0,(Americhernus:94.0,Artemia:89.0)HTU44:15.0)HTU39:13.0)HTU49:9.5,(Vonones:68.0,(Amblypygid:70.0,(((Centruoides:67.5,(Hadrurus:69.5,Pauroctonus:60.5)HTU50:13.5)HTU37:10.0,(Scaphopods:82.0,((Lymnaea_stagnalis:76.0,Mastigoproctus:61.0)HTU42:9.0,((Aplysia_californica:85.0,(Gea:83.0,Hypochilus:58.0)HTU36:3.0)HTU55:4.0,(Siphonaria_pectinata:84.5,Thelichoris:57.5)HTU62:6.5)HTU43:24.0)HTU41:11.0)HTU57:3.5)HTU40:7.0,((Ilyanassa_obsoleta:77.0,(Perotrochus_lucaya:75.0,(Chanbria:61.0,Littorina_littorea:75.5)HTU47:3.0)HTU46:2.5)HTU45:12.0,(Lottia_gigantea:80.0,(Crassostrea_virginica:113.5,(Limulus:62.0,(Yoldia_limatula:79.5,(Mytilus_californianus:84.0,(Lingula_anatina:81.5,(Venerupis_decussatus:109.5,Venerupis_philippinarum:82.0)HTU54:6.5)HTU61:7.5)HTU60:20.0)HTU53:3.5)HTU59:7.0)HTU58:6.0)HTU56:6.0)HTU51:8.5)HTU38:14.0)HTU48:7.5)HTU52:11.0)HTU35:22.0)HTU34:4.0)HTU33:2.5)HTU32[571.0];
